Src/AutoNSubstituteUnitTest/AutoConfiguredFixtureIntegrationTest.cs

```diff
@@ -427,23 +427,5 @@ public void InterfaceShadowedMethodsReturnValueFromFixture()
             Assert.Same(frozenString, (result as IInterfaceWithShadowedMethod).Method(0));
             // Teardown
         }
-
-        [Fact]
-        public void InterfacesImplementingIEnumerableReturnFiniteSequence()
```

Comment:
My reasoning to delete it was that, now that `Fixture` is able to create well-behaved enumerators on its own, this test only checks that `IMyList<T>.GetEnumerator` is correctly setup by the `AutoConfiguredNSubstituteCustomization`. Since there's nothing special about the method itself - `IMyList<T>.GetEnumerator` is just another method - this behavior is already covered by other tests.

But I can also see why keeping the test in would help as a piece of documentation and as a functional/regression test as well.

Whichever way you wanna go, I'll be happy to do it.